### ISSUE TITLE
Incorrect lr_mult value when using find_generator

### ISSUE THREAD

**Issue by JonnoFTW** [surmenok/keras_lr_finder#18]
When using `find_generator`:

https://github.com/surmenok/keras_lr_finder/blob/7870043d86101010b2a644df22b4018af58534fb/keras_lr_finder/lr_finder.py#L76

`lr_mult` causes the learning rate to converge at `end_lr` after only 1 epoch. If you want to use 2 or more epochs, then your learning rate will exceed `end_lr` and end up training on a learning rate that is far too high than intended.

The fix here is (when `end_lr=1,start_lr=0.001,steps_per_epoch=1000,epochs=4`):

```python
lr_mult = ((float(end_lr) / float(start_lr)) ** (1. / float(steps_per_epoch*epochs)))
```
Here's a sample script to show how the two functions diverge `while cur_lr <= end_lr`
![Figure_1](https://user-images.githubusercontent.com/650314/61192588-9693a400-a6f4-11e9-94eb-3493240cab95.png)

**Comment by surmenok** [comment]
Nice catch, thank you! Would you mind to create a PR to fix this?

**Comment by JonnoFTW** [comment]
Will do, I'll update the documentation as well with some better examples
